Sources/Spezi/Spezi.docc/Module/Interactions with SwiftUI.md

```diff
@@ -0,0 +1,106 @@
+# Interactions with SwiftUI
+
+Interact with the SwiftUI view hierarchy and its environment.
+
+<!--
+
+This source file is part of the Stanford Spezi open-source project
+
+SPDX-FileCopyrightText: 2023 Stanford University and the project authors (see CONTRIBUTORS.md)
+
+SPDX-License-Identifier: MIT
+
+-->
+
+## Overview
+
+A `Module` has several ways of interacting with the SwiftUI view hierarchy and its environment.
+
+### Managing Model state
+
+By using the ``Module/Model`` property wrapper, your `Module` can place an [@Observable](https://developer.apple.com/documentation/observation/observable())
+model type into the global SwiftUI view environment.
+
+Below is a short code example that demonstrates this functionality:
+```swift
+@Observable
+class ExampleModel { // your model type you want to make accessible
+    var someState: Bool = false
+
+    init() {}
+}
+
+
+class ExampleModel: Module { // your model the app configures
+    @Model var model = ExampleModel()
+}
+
+
+class ContentView: Module { // access the model within SwiftUI
+    @Environment(ExampleModel.self) var model
+}
```

Comment:
I would suggest to maybe split up this code example into smaller blocks with a smaller text between them instead of using the comments here. And Module -> View.
```suggestion
class ContentView: View { // access the model within SwiftUI
    @Environment(ExampleModel.self) var model
}
```

Reply:
good catch 🚀